### DISCUSSION TITLE
Sub Thumbnails need a proper UI Background:

### DISCUSSION THREAD

**Discussion by aeldred28** [FakeFishGames/Barotrauma#8514] [Ideas and suggestions]
Something that has always bothered me about the UI unification in the game is the Sub menu selection;

All the UI elements in the menu share a clean green texture/color that fit together, with exception of the submarine renders which are just a flat blue color, reminiscent of a missing texture file.

It feels like a broken UI element, as if part of the UI didn't download correctly. To fix this problem I want to share with the devs the solution that I gave all my custom subs:

![image](https://user-images.githubusercontent.com/75105816/158036594-0922e59b-e44d-4f2a-8e9b-996860cbde6b.png)
^This is the vanilla sub screen with the blue background 

![image](https://user-images.githubusercontent.com/75105816/158036680-f3d19d8b-2079-4778-90a0-2edb69035bd2.png)
^This is my solution with a green texture that has a light green border the same as the other elements. (forgive the banding, it would be easily solved).

![2000x2000 Template](https://user-images.githubusercontent.com/75105816/158036860-80c9ae84-aa1a-43c5-bb13-c3bf1dd810c4.jpg)
^This is the texture I used. (Forgive the banding and compression for this upload)

I hope the devs will give this area some attention for an upcoming update, and consider offering their thoughts on the proposal.

**Comment by Udrakan** [comment]
The New Game menu in general needs an upgrade, but this is a good start. Vanilla BG for subs doesn't indeed look proper, as if the background was supposed to be blue to show a schematic, but instead the sub is shown as in-game. Either a schematic, or what Scale has sent would work.

**Comment by Working-Joe** [comment]
Do not worry about the compression, I've seen what baro does to textures

On a more serious note, it's also nice if there is some contrast with the background so that it's easy to make out the contours
